modules/ROOT/partials/commands/advtable-cmds.adoc

```diff
@@ -1,11 +1,12 @@
+[cols="d,a"]
```

Comment:
Why was this added? From what I can see it shouldn't have been required. If it's for the admonition include that's probably better off being left as plain text as it takes up more space otherwise (and plain text is more consistent with the [current docs](https://www.tiny.cloud/docs/plugins/premium/advtable/#commands)). For example:
![image](https://user-images.githubusercontent.com/1575550/159799164-18c5dff9-3edf-414c-8a63-d24ad6b5825e.png)
vs
![image](https://user-images.githubusercontent.com/1575550/159799204-e5af4df9-4951-44cd-91fe-e3a451e81c3f.png)

Reply:
Oh so you want the admonition to render as a text and not like the other admonitions ? so I should revert the change ?

Reply:
Yes, that's correct... that's what it was likely previously in the old docs and is also the default AsciiDoc behaviour and makes some sense because as I said the full admonition is bulky. I'm still not confident, but given the goal of this is to keep it similar to the old version I think leaving it as text is better option here. I'd suggest noting this on the Notion page as something to look at later and for now just leave it as text.

Reply:
Done